README.md

```diff
@@ -57,158 +57,167 @@ To run the Quart application correctly, it requires some environment variables t
-
-- **Description:** Health check message for the server.
-- **Example:** `MEMBRANE_HEALTH_MESSAGE=ok`
-
-### MEMBRANE_FRONTEND
+#### MEMBRANE_FRONTEND
 
 - **Description:** Redirect URL leading users to the login frontend.
 - **Example:** `MEMBRANE_FRONTEND=http://localhost:3000`
 
-### MEMBRANE_SECRET_KEY
+#### MEMBRANE_SECRET_KEY
 
 - **Description:** The secret key used for creating encrypted tokens.
 - **Example:** `MEMBRANE_SECRET_KEY=your_secret_key`
 
-### MEMBRANE_CLIENT_PUBLIC_KEYS_DIRECTORY
+#### MEMBRANE_CLIENT_PUBLIC_KEYS_DIRECTORY
 
 - **Description:** Path to the directory where client public keys are stored for JWT validation.
 - **Example:** `MEMBRANE_CLIENT_PUBLIC_KEYS_DIRECTORY=keys/`
 
-### MEMBRANE_SERVER_PRIVATE_KEY
+#### MEMBRANE_SERVER_PRIVATE_KEY
 
 - **Description:** Path to the server's private key file used for creating and signing tokens.
 - **Example:** `MEMBRANE_SERVER_PRIVATE_KEY=keys/server_private_key.pem`
 
-### MEMBRANE_SERVER_PUBLIC_KEY
+#### MEMBRANE_SERVER_PUBLIC_KEY
 
 - **Description:** Path to the server's public key file used for verifying tokens.
 - **Example:** `MEMBRANE_SERVER_PUBLIC_KEY=keys/server_public_key.pem`
 
-### MEMBRANE_TEST_APP_PUBLIC_KEY
+#### MEMBRANE_COMM_CONNECTION_STRING
+
+- **Description:** Connection string for the Azure communication service.
+- **Example:** `MEMBRANE_COMM_CONNECTION_STRING=your_azure_communication_service_connection_string`
 
-- **Description:** Path to the test application's public key.
-- **Example:** `MEMBRANE_TEST_APP_PUBLIC_KEY=keys/testapp1_public_key.pem`
+#### MEMBRANE_SENDER_EMAIL
 
-### MEMBRANE_TEST_APP_PRIVATE_KEY
+- **Description:** Email address that will send emails.
+- **Example:** `MEMBRANE_SENDER_EMAIL=DoNotReply@your_domain.com`
 
-- **Description:** Path to the test application's private key.
-- **Example:** `MEMBRANE_TEST_APP_PRIVATE_KEY=keys/testapp1_private_key.pem`
+### Optional Variables
 
-### MEMBRANE_JWT_ACCESS_TOKEN_EXPIRE_SECONDS
+#### MEMBRANE_JWT_ACCESS_TOKEN_EXPIRE_SECONDS
 
 - **Description:** Expiration time (in seconds) for the JWT access token.
 - **Example:** `MEMBRANE_JWT_ACCESS_TOKEN_EXPIRE_SECONDS=300`
 
-### MEMBRANE_JWT_EXPIRE_SECONDS
+#### MEMBRANE_JWT_EXPIRE_SECONDS
 
 - **Description:** General JWT expiration time in seconds.
 - **Example:** `MEMBRANE_JWT_EXPIRE_SECONDS=300`
 
-### MEMBRANE_SESSION_LIFETIME_SECONDS
+#### MEMBRANE_SESSION_LIFETIME_SECONDS
 
 - **Description:** Duration (in seconds) after which the session will expire.
 - **Example:** `MEMBRANE_SESSION_LIFETIME_SECONDS=300`
 
-### MEMBRANE_SESSION_COOKIE_SECURE
+#### MEMBRANE_SESSION_COOKIE_SECURE
 
 - **Description:** Indicates if the session cookie should be secure.
 - **Example:** `MEMBRANE_SESSION_COOKIE_SECURE=true`
 
-### MEMBRANE_SESSION_TYPE
+#### MEMBRANE_SESSION_TYPE
 
 - **Description:** Specifies the storage for session data. Options: 'filesystem', 'redis', 'memcached', etc.
 - **Example:** `MEMBRANE_SESSION_TYPE=null`
```

Comment:
Can you go back through the environment variable and add a 

**Reference: https://flask-session.readthedocs.io/en/latest/config.html

with a link to the subsystem doc that uses this value? It occurs to me the description is not sufficient to decide the value to use and I was unaware that null was a valid value for example.

Reply:
Done

Reply:
Repeat adding reference for all variables that feed dependency libraries.

Reply:
Done